processor/compile/html-block.ts

```diff
@@ -1,9 +1,10 @@
 import { HTMLBlock } from '../../types';
+import { reformatHTML, getHProps } from '../utils'
 
 const htmlBlock = (node: HTMLBlock) => {
-  const html = node.data.hProperties.html;
-  const attributes = Object.keys(node.data?.hProperties).map(key => `${key}="${node.data?.hProperties[key]}"`).join(' ')
-  return `<HTMLBlock${attributes && ' ' + attributes}>{\`${ html }\`}</HTMLBlock>`;
+  const { runScripts, html } = getHProps<HTMLBlock['data']['hProperties']>(node);
+
+  return `<HTMLBlock${runScripts != null ? ' runScripts="' + runScripts + '"' : ''}>{\`\n${ reformatHTML(html) }\n\`}</HTMLBlock>`;
```

Comment:
If we're converting this to use a template string, we should probably also make this easier to read while we're at it:

```suggestion
  return `<HTMLBlock ${runScripts != null ? `runScripts="${runScripts}"` : ''}>{\`
  ${reformatHTML(html)}
\`}</HTMLBlock>`;
```

Reply:
sure! although i'm aesthetically offended by the extra space that's left if there's no `runScripts` prop 😂